runtime/kusama/src/xcm_config.rs

```diff
@@ -326,6 +326,14 @@ impl Contains<RuntimeCall> for SafeCallFilter {
 				pallet_nomination_pools::Call::update_roles { .. } |
 				pallet_nomination_pools::Call::chill { .. },
 			) |
+			RuntimeCall::Hrmp(..) |
+			RuntimeCall::Registrar(
+				paras_registrar::Call::deregister { .. } |
```

Comment:
Being here is just about "we consider the weight of these functions okay", right? there's still barriers and more to prevent actual execution based on chain?

Reply:
Yes, there is also the extrinsic itself that checks whether the FRAME origin is what it expects.